Composer/packages/lib/indexers/src/types/bf-lu.d.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-declare module 'ludown' {
+declare module '@bfcomposer/bf-lu/lib/parser' {
```

Comment:
Since this is a typescript package, it should ship with its own declaration files. Let's make sure that is the case and remove this.

Reply:
AFAIK, bf-lu is javascript, the whole bf-cli is js package, and don't have a priority of converting to ts, which is something i proposed multiple times. So, we probably still need this.